packages/mint-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.4.2",
+  "version": "1.5.1",
```

Comment:
Make sure to keep an eye on #265, if that goes out then this number will change again

Reply:
I used 1.5.2 in anticipation of 1.5.1 being release first. I will wait for that release for a final rebase before asking for another PR approval